src/kvstore/RocksEngine.h

```diff
@@ -70,6 +90,11 @@ class RocksPrefixIter : public KVIterator {
   RocksPrefixIter(std::unique_ptr<rocksdb::Iterator> iter, rocksdb::Slice prefix)
       : iter_(std::move(iter)), prefix_(prefix) {}
 
+  explicit RocksPrefixIter(rocksdb::Slice prefix) : prefix_(prefix) {
+    iterateUpperBoundKey_ = NebulaKeyUtils::lastKey(prefix_.ToString(), 128);
```

Comment:
Here you are padding fixed 128 EOF. Is there any performance impact when the prefix length is already very long?

Reply:
128 is referred to the existing code in Part::cleanup, as far as I can see, there is no performance degradation.